pkg/scheduler/framework/plugins/podtopologyspread/filtering_test.go

```diff
@@ -1299,6 +1299,45 @@ func TestPreFilterState(t *testing.T) {
 			},
 			enableMatchLabelKeys: true,
 		},
+		{
+			name: "key in matchLabelKeys is ignored when LabelSelector is nil",
```

Comment:
Could you add the test case "no match with any Pods when LabelSelector is nil"? (not directly related to MatchLabelKeys though)

Reply:
Done